components/switch/w-switch.vue

```diff
@@ -1,42 +1,57 @@
+<script setup>
+import { computed, ref } from 'vue'
+import { switchToggle as ccSwitch } from '@warp-ds/component-classes'
+import { createModel, modelProps } from 'create-v-model'
+import { id } from '#util'
+
+const p = defineProps({
```

Comment:
should we have `props` instead of `p` for readability?

Reply:
Using p for props is quite a de facto standard I believe. 😄 I am not sure who I asked about this before... but I think there are a few more components that uses p?

Reply:
But it would be a good thing to have it the same everywhere, whatever we choose. 👍

Reply:
I want to remember that I asked you Magnus if `p` was a some sort of Vue non-official standard and I want to remember you said yes 😁

Reply:
aha okay, there are many components that use `props` as well 😅 then it doesn't matter for now, but we should do the same everywhere later